.github/workflows/http-dotnet-httpclient.yml

```diff
@@ -20,8 +21,10 @@ jobs:
         uses: actions/setup-dotnet@v1.9.0
         with:
           dotnet-version: 6.0.x
-      - run: ./scripts/updateNugetCredentials.ps1 -username "${{ secrets.PUBLISH_GH_USERNAME }}" -apiToken "${{ secrets.PUBLISH_GH_TOKEN }}" -nugetFileAbsolutePath "${{ env.relativePath }}/nuget.config"
-        shell: pwsh
+      - name: Add NuGet kiota source
+        # NOTE: Password encryption is not supported for the linux platform (Encryption is only supported on Windows platforms.)
+        run: |
+          dotnet nuget add source ${{env.NUGET_URL}} -n github -u ${{secrets.PUBLISH_GH_USERNAME}} -p ${{secrets.PUBLISH_GH_TOKEN}} --store-password-in-clear-text
```

Comment:
The step on line 55 will also need to be cleaned up as well as the workflow fails.